### PR TITLE
chore: adding resource links to readme

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

It was not easy to find certain things, so I've added them to the readme:
* the In the Wild page, and other markdown files on the repo that have no links to them
* Instructions to _actually join_ the dev mailing list
* The superset wiki
* The database rolodex doc
* The SIP board
* The community calendar

Maybe this'll boost some visibility/engagement?

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

n/a

### TESTING INSTRUCTIONS

read, and click links ;)

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by sfirke — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -138,7 +138,9 @@ Want to add support for your datastore or data engine? Read more [here](https://
...
 - [Join our community's Slack](http://bit.ly/join-superset-slack)
   and please read our [Slack Community Guidelines](https://github.com/apache/superset/blob/master/CODE_OF_CONDUCT.md#slack-community-guidelines)
-- [Join our dev@superset.apache.org Mailing list](https://lists.apache.org/list.html?dev@superset.apache.org)
+- [Join our dev@superset.apache.org Mailing list](https://lists.apache.org/list.html?dev@superset.apache.org). To join, simply send an email to [dev-subscribe@superset.apache.org](mailto:dev-subscribe@superset.apache.org)
+- If you want to help troubleshoot GitHub Issues involving the numerous database drivers that Superset supports, please consider adding your name and the databases you have access to on the [Superset Database Familiarity Rolodex](https://docs.google.com/spreadsheets/d/1U1qxiLvOX0kBTUGME1AHHi6Ywel6ECF8xk_Qy-V9R8c/edit#gid=0)
+- Join Superset's Town Hall and [Operational Model](https://preset.io/blog/the-superset-operational-model-wants-you/) syncs, all of which are available on the [Superset Community Calendar](https://superset.apache.org/community)
```
[the remark this answers is not in the document]
Comment (sfirke) [reply README.md:143]: I thought "sync" as a noun to mean "meeting" was jargony 🦒

**Comment by sfirke** [comment]
nice additions, thanks Evan!

File: README.md [hunk 2]
```diff
@@ -149,6 +151,12 @@ how to set up a development environment.
 
 ## Resources
 
+- [Superset "In the Wild"](RESOURCES/INTHEWILD.md) - open a PR to add your org to the list!
+- [Feature Flags](RESOURCES/FEATURE_FLAGS.md) - the status of Superset's Feature Flags.
+- [Standard Roles](RESOURCES/STANDARD_ROLES.md) - How RBAC permissions map to roles.
+- [Superset Wiki](https://github.com/apache/superset/wiki) - Tons of additional community resources: best practices, community content and other information.
+- [Superset SIPs](https://github.com/orgs/apache/projects/170) - The status of Superset's SIPs (Superset Improvment Proposals) for both consensus and implementation status.
+
 Superset 2.0!
```
[the remark this answers is not in the document]
Comment (rusackas) [reply README.md:160]: I think I'll actually put all of these on the Wiki in an orderly fashion and delete these, in a subsequent PR.

[the remark this answers is not in the document]
Comment (rusackas) [reply README.md:160]: Maintaining the Wiki (and directing traffic to it) is MUCH easier than opening PRs for every webinar link ;)

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rusackas) [reply README.md:143]: Jargony?! But it's SO important to let people know we're driving a new paradigm to forge process synergies, with cross-functional holistic thinking and transformative agile methodologies!

**Review by sfirke — APPROVED** [review]

**Pull request merged**